device/bgp/evpn/bgpv6evpnethernetsegment.yaml

```diff
@@ -0,0 +1,24 @@
+components:
```

Comment:
The contents of this file are identical to that of bgpv4evpnethernetsegment.yaml. Why do we have separate models for v4 and v6 if the contents are the same?

Reply:
Currently there is no difference as we are providing only basic support and for PMSI type Ingress replication only. But later on broadcast domain v4 and v6 will have v4 and v6 addresses for Group address and  SenderAddress/P Root node address when PIM comes as PMSI tunnel option. Keeping that it mind we have kept both v4 and v6 separate for future compatibility so that the respective attributes can be put in the respective v4/v6 class. Please note that this class hierarchy doesn't impact the overall OTG modelling . Its just internal to the implementation.

v4peer->v4evpnethernetsegment->v4evpnevpnevivxlan->v4evpnbroadcastdomainvxlan
v4peer->v6evpnethernetsegment->v6evpnevpnevivxlan->v6evpnbroadcastdomainvxlan